src/features/organizations/hooks/useUpcomingOrgEvents.ts

```diff
@@ -20,3 +20,17 @@ export default function useUpcomingOrgEvents(orgId: number): ZetkinEvent[] {
       ),
   });
 }
+
+export function useOrgEvents(orgId: number): ZetkinEvent[] {
```

Comment:
Why two hooks in the same module?

Reply:
I have no idea this must have happened without me seeing it when merging prs during code camp. i think there might be duplicated logic here as well